pom.xml

```diff
@@ -341,10 +341,10 @@
 
         <!-- Asciidoctor -->
 
-        <version.asciidoctor.plugin>2.2.6</version.asciidoctor.plugin>
```

Comment:
Whoa there, maybe we should check the documentation output on this one :)

Reply:
yeah 😄 I was just building it while a new mandrel image is downloading 🙈 
pdf and HTML are there and even have content in them 😃

Reply:
Thanks for checking!